manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -53,6 +53,13 @@ pilot:
 
   plugins: []
 
+  # The time interval if no activity on the connection it pings the peer to see if the transport is alive
+  keepaliveInterval: 30s
+
+  # After having pinged for keepalive check, the client/server waits for a duration of keepaliveTimeout
+  # and if no activity is seen even after that the connection is closed.
+  keepaliveTimeout: 10s
+
```

Comment:
Given the if conditional above, I'd prefer we either (1) comment these out or (2) remove them. I'm fine with them as hidden values in the helm chart.

Reply:
Started going with 2, but then a `make gen` removes references from `gen-istio.yaml` and started feeling awkward compared to having it explicitly defined in the operator package.